### ISSUE TITLE
funny floating thorianite

### ISSUE THREAD

**Issue by deltanedas** [Regalis11/Barotrauma#10240]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

when doing abyss mining i regularly find bits of thorianite, never any other ore for some reason, floating either close to the rock or far out

![screenshot](https://user-images.githubusercontent.com/39013340/200059710-0d563f0a-5a3c-4568-8a46-240b95051dea.png)

### Reproduction steps

1. go to abyss rock
2. see it sometimes

### Bug prevalence

Happens every now and then

### Version

0.19.14.0

### Which operating system did you encounter this bug on?

Linux

**Comment by Regalis11** [comment]
Thank you for the report! I'm afraid this might be next to impossible to reproduce without a save file or a level seed. :/

**Comment by Regalis11** [comment]
Thank you for the report! I gave this a quick test, and couldn't find any out-of-place thorianite bits. I also noticed the thorianite in your pic doesn't look the same as the vanilla thorianite - I suppose you're using some mod that modifies the minerals? If so, I'd suspect this is an issue in the mod, not the vanilla game.

![image](https://user-images.githubusercontent.com/14351070/200119078-8761c257-14bc-49cc-b4d9-7318d9e9353e.png)
